Makefile

```diff
@@ -174,7 +174,7 @@ itest-race: build-itest-race itest-only
 itest-parallel: build-itest db-instance
 	@$(call print, "Running tests")
 	rm -rf itest/*.log itest/.logs-*; date
-	EXEC_SUFFIX=$(EXEC_SUFFIX) echo "$$(seq 0 $$(expr $(ITEST_PARALLELISM) - 1))" | xargs -P $(ITEST_PARALLELISM) -n 1 -I {} scripts/itest_part.sh {} $(NUM_ITEST_TRANCHES) $(TEST_FLAGS) $(ITEST_FLAGS)
+	EXEC_SUFFIX=$(EXEC_SUFFIX) scripts/itest_parallel.sh $(ITEST_PARALLELISM) $(NUM_ITEST_TRANCHES) $(TEST_FLAGS) $(ITEST_FLAGS)
```

Comment:
I think we could move the EXIT_STATUS to an argument as well.

Reply:
what `EXIT_STATUS`?

Reply:
Sorry, I meant `EXEC_SUFFIX` :see_no_evil:

Reply:
Cool moved!

Reply:
hmm actually i don't think it can be passed in as a positional argument since it's only set for windows...

Reply:
Ah, true. I guess double quotes around it would've solved that, but let's keep it the way it is now.